api/src/main/java/org/pmiops/workbench/db/model/DataSet.java

```diff
@@ -18,6 +19,7 @@
 @Entity
 @Table(name = "data_set")
 public class DataSet {
```

Comment:
This value class doesn't have `equals()` or `hashCode()` defined, which could bite you if you try to use it as a map key or set element.

Reply:
Let me add a `.equals()`

Reply:
IJ can give you one for free.

Additionally, would a copy constructor in this class help you with implementing `cloneDataSetToWorkspace()`?